tasks/examples.py

```diff
@@ -56,6 +57,51 @@ def build(ctx, clean=False):
     )
 
 
+@task()
+def build_mpi(ctx, clean=False):
```

Comment:
Let's move this to a new `tasks/native_mpi.py` as it's not really related to the examples